src/Microsoft.ML.Transforms/Text/TextCatalog.cs

```diff
@@ -20,34 +20,32 @@ public static class TextCatalog
         /// <param name="catalog">The text-related transform's catalog.</param>
         /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.</param>
         /// <param name="inputColumnName">Name of the column to transform. If set to <see langword="null"/>, the value of the <paramref name="outputColumnName"/> will be used as source.</param>
-        /// <param name="advancedSettings">Advanced transform settings</param>
-        /// <example>
-        /// <format type="text/markdown">
-        /// <![CDATA[
-        /// [!code-csharp[FeaturizeText](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/TextTransform.cs)]
```

Comment:
>TextTransform [](start = 101, length = 13)

you removing this example becaue it doesn't use this exact method?

Reply:
yeap. removed it from here, and added to the API below  (this was one of Senja's comments)

---
In reply to: [254802756](https://github.com/dotnet/machinelearning/pull/2394#discussion_r254802756) [](ancestors = 254802756)